spring-cloud-data-rest/pom.xml

```diff
@@ -80,6 +82,25 @@
 			<plugin>
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
+				<configuration>
+					<classifier>exec</classifier>
```

Comment:
It was there for the module samples originally because I wondered if it might be useful to have the module as a thin jar for composition. I don't know what relevance that has here.